### PR TITLE
New package: Max.Backup version 0.1.3

### PR DESCRIPTION
**Pull request by kellyelton**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/118640)

### CODE REVIEW

File: manifests/m/Max/Backup/0.1.3/Max.Backup.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Max.Backup
+PackageVersion: 0.1.3
+PackageLocale: en-US
+Publisher: Kelly Elton
+PublisherUrl: https://github.com/kellyelton
+PublisherSupportUrl: https://github.com/kellyelton/MaxBackup/issues
+Author: Kelly Elton
+PackageName: max backup
```
Comment (stephengillie) [inline manifests/m/Max/Backup/0.1.3/Max.Backup.locale.en-US.yaml:11]: Should this be capitalized and joined?
```suggestion
PackageName: MaxBackup
```

File: manifests/m/Max/Backup/0.1.3/Max.Backup.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Max.Backup
```
Comment (stephengillie) [inline manifests/m/Max/Backup/0.1.3/Max.Backup.installer.yaml:4]: Hi @kellyelton,

The PackageIdentifier should be in form `Author.Package`: 

```suggestion
PackageIdentifier: kellyelton.MaxBackup
```

Does this look okay to you? 
Please remember to update the file names and paths, and other PackageIdentifiers, to match.

File: manifests/m/Max/Backup/0.1.3/Max.Backup.locale.en-US.yaml (hunk 1 above)
Comment (kellyelton) [reply manifests/m/Max/Backup/0.1.3/Max.Backup.locale.en-US.yaml:11]: is that required?

Comment (kellyelton) [reply manifests/m/Max/Backup/0.1.3/Max.Backup.locale.en-US.yaml]: I've made the requested change.

**Review by kellyelton — COMMENTED** [review]
I've made the requested changes

**Pull request closed without merging**